### PR TITLE
Private local testnet support

### PR DESCRIPTION
**Pull request by kantai**
These changes were required for supporting a private local testnet with the latest version of bitcoind

### CODE REVIEW

File: src/burnchains/bitcoin/indexer.rs
Comment (psq) [inline src/burnchains/bitcoin/indexer.rs:503]: network

File: testnet/stacks-node/src/neon_node.rs
Comment (jcnelson) [inline testnet/stacks-node/src/neon_node.rs:1439]: Is this error reachable without being indicative of database corruption?  The only way I can think it could be reached is if you're mining off of the genesis block, which has no parent.

Comment (kantai) [reply testnet/stacks-node/src/neon_node.rs:1439]: I think that's correct, but this replicates the previous behavior of the miner (which would also just "not mine" rather than panic here).

**Review by pavitthrap — APPROVED** [review]
LGTM!

**Review by jcnelson — APPROVED** [review]
LGTM! Let's get this tested on the testnet before merging to master.

**Comment by kantai** [comment]
> LGTM! Let's get this tested on the testnet before merging to master.

Yep -- will hold off on merging until 2.0.11.1.0 is released

**Pull request merged**
